components/builder-jobsrv/src/data_store.rs

```diff
@@ -165,7 +165,7 @@ impl DataStore {
                                         FOR UPDATE SKIP LOCKED
                                         LIMIT $1
                                     LOOP
-                                        UPDATE jobs SET job_state='Dispatched', updated_at=now() WHERE id=r.id RETURNING * INTO r;
+                                        UPDATE jobs SET job_state='Dispatched', scheduler_sync=false, updated_at=now() WHERE id=r.id RETURNING * INTO r;
```

Comment:
This particular change is not critical, it's a nice to have, re-creating the jobsrv db before we go live to pick it up is fine.